features/elastictranscoder/step_definitions/elastictranscoder.js

```diff
@@ -1,11 +1,11 @@
-const { ElasticTranscoder } = require("../../../clients/client-elastic-transcoder");
 const { S3 } = require("../../../clients/client-s3");
 const { IAM } = require("../../../clients/client-iam");
```

Comment:
Move S3 and IAM requires inside Before tag?

Reply:
Thank you! very nice catch!